sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
```

Comment:
`app/config.php` is always included and all the content is global throughout the application. Just add a array `$deplyment_settings = array('missing_persons' => true);` there and access it in this library. Everything else remains same.

Anyway, at the time of deployment `config.php` has to be changed.